### PR TITLE
add serve for vue/laravel 

### LINKED ISSUE #55: Start dev server for Vue and Laravel projects
Laravel: php artisan serve
Vue: npm run serve

### PR DESCRIPTION
**Pull request by Rexogamer**
Closes #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,14 @@ Contact us [on Discord.](https://discord.gg/eqWstJu)
...
-#### Vue
-- Vue CLI
 #### Laravel
 - Laravel
 - Composer
 - PHP
+- serve
+#### Vue
+- Vue CLI
+- serve
```
Comment (jarodburchill) [inline README.md:19]: no

File: README.md [hunk 2]
```diff
@@ -9,12 +9,14 @@ Contact us [on Discord.](https://discord.gg/eqWstJu)
 - A text editor i.e. VS Code, Atom, etc. (Optional but recommended.)
 #### React/Node.js/Express.js
 - npm
-#### Vue
-- Vue CLI
 #### Laravel
 - Laravel
 - Composer
 - PHP
+- serve
```
Comment (jarodburchill) [inline README.md:16]: no

File: project_types.py [hunk 3]
```diff
@@ -58,11 +58,13 @@ def Express():
 def Laravel():
     subprocess.check_call("laravel new {}".format(projectName), shell=True)
     os.chdir(projectName)
+    subprocess.check_call("php artisan serve", shell=True)
```
Comment (jarodburchill) [inline project_types.py:61]: This is not where we do the serve.... See where we do it for the React project types in the automate_project.py file.

File: project_types.py [hunk 4]
```diff
@@ -58,11 +58,13 @@ def Express():
...
 
 
 def Vue():
     subprocess.check_call("vue create {}".format(projectName), shell=True)
     os.chdir(projectName)
+    subprocess.check_call("npm run serve", shell=True)
```
Comment (jarodburchill) [inline project_types.py:67]: This is not where we do the serve.... See where we do it for the React project types in the automate_project.py file.

**Review by jarodburchill — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Rexogamer) [reply README.md:19]: how very helpful of you, I’ll take your very detailed feedback into account!

Comment (jarodburchill) [reply README.md:19]: It is not a requirement

**Pull request closed without merging**